grails-app/migrations/core/2020-09-18-convert-API-keys-to-Ethereum-IDs.groovy

```diff
@@ -0,0 +1,122 @@
+package core
+
+import com.unifina.domain.IntegrationKey
+import com.unifina.domain.SignupMethod
+import com.unifina.domain.Stream
+import com.unifina.domain.ExampleType
+import com.unifina.domain.Permission
+import com.unifina.utils.IdGenerator
+import com.unifina.security.ApiKeyConverter
+import com.unifina.service.EthereumIntegrationKeyService
+import com.unifina.utils.AlphanumericStringGenerator
+import grails.converters.JSON
+import groovy.sql.Sql
+
+def getAccountAddress(String apiKeyId) {
+	String privateKey = ApiKeyConverter.createEthereumPrivateKey(apiKeyId)
+	return "0x" + EthereumIntegrationKeyService.getAddress(privateKey)
+}
+
+def createIntegrationKey(String integrationKeyName, String accountAddress, Integer userId, Sql sql) {
+	Date now = new Date()
+	return sql.executeInsert("INSERT INTO integration_key (id, version, name, json, user_id, service, id_in_service, date_created, last_updated) VALUE (?, ?, ?, ?, ?, ?, ?, ?, ?)", [
+		IdGenerator.get(),
+		0,
+		integrationKeyName,
+		([ address: accountAddress ] as JSON).toString(),
+		userId,
+		IntegrationKey.Service.ETHEREUM_ID.name(),
+		accountAddress,
+		now,
+		now
+	]);
+}
+
+def createInboxStream(long userId, String accountAddress, Sql sql) {
```

Comment:
Hmm. Where did this inbox stream thing come from? It doesn't exist anymore, shouldn't be needed.

Reply:
Discussed earlier with Eric and understood from that discussion that it would be best to add inbox streams. But if they are not needed anymore, I'll remove this functionality here.